crawl/workspace.py

```diff
@@ -21,7 +21,7 @@ class Workspace:
     Maven concepts.
     """
     def __init__(self, repo_root_path, config, maven_install_info,
-                 pom_content, dependency_metadata, verbose=False):
+                 pom_content, dependency_metadata, override_file_info = [], verbose=False):
```

Comment:
Can't we get override_file_info from the config?  The config is passed into this ctor.